### DISCUSSION TITLE
Port to the AtariVCS?

### DISCUSSION THREAD

**Discussion by h3rb** [paladin-t/paladin-t.github.io#4] [Ideas]
It would be neat to see this (Basic8) ported to the AtariVCS

You should contact Atari (hellovcs@atari.com) about publishing, they need new apps for Atari and would give you a chance

**Comment by paladin-t** [comment]
Didn't know that, it's good to see Atari is back to the game console industry, and the console looks awesome. I will contact them for a possible opportunity.
